### PR TITLE
Fix code style violations reported by new VS linter

### PR DESCRIPTION
**Pull request by hnrqbaggio**
Renamed variables, types and namespace to better match the UE naming style.
Added `.editorconfig` to enforce checks using the new VS feature in preview.

### CODE REVIEW

File: Source/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+cpp_naming_rule.structs_prefixed.style						= unreal_engine_structs
+
+cpp_naming_rule.enums_prefixed.symbols						= enums
+cpp_naming_rule.enums_prefixed.style						= unreal_engine_enums
+
+cpp_naming_rule.templates_prefixed.symbols					= templates
+cpp_naming_rule.templates_prefixed.style					= unreal_engine_templates
+
+cpp_naming_rule.general_names.symbols						= all_symbols
+cpp_naming_rule.general_names.style							= unreal_engine_default
```
Comment (RaulPerez1) [inline Source/.editorconfig:27]: If this is meant to be all aligned this and quite a few others below are showing misaligned.

File: Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp [hunk 2]
```diff
@@ -185,21 +185,21 @@ struct AssetIndex
 				Classes.Add(OwnerName).Class = Owner;
 			}
 
-			ClassEntry& entry = Classes[OwnerName];
+			FClassEntry& Entry = Classes[OwnerName];
```
Comment (RaulPerez1) [inline Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp:188]: Which rule is capitalizing Entry?

**Review by martin-toman — APPROVED** [review]

File: Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp (hunk 2 above)
Comment (hnrqbaggio) [reply Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp:188]: The one in line 26 of the .editorconfig. 

`general_names.style.capitalization` = pascal_case
`general_names.symbols.applicable_kinds` = *

All identifier names are pascal cased by the UE code guidelines. Not something I personally find pretty, but we don't make up the rules :)

Comment (RaulPerez1) [reply Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp:188]: Totally my bad, I saw pascal case and for some reason mentally mapped it to camel case...

**Review by RaulPerez1 — APPROVED** [review]

**Pull request merged**